bigtable-hbase-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableOptionsFactory.java

```diff
@@ -216,6 +216,19 @@
   public static final String BIGTABLE_BUFFERED_MUTATOR_MAX_MEMORY_KEY =
       "google.bigtable.buffered.mutator.max.memory";
 
+  /**
+   * Turn on a feature that will reduce the likelihood of BufferedMutator overloading a Cloud
+   * Bigtable cluster.
+   */
+  public static final String BIGTABLE_BUFFERED_MUTATOR_ENABLE_THROTTLING =
+      "google.bigtable.buffered.mutator.enable.throttling";
```

Comment:
nit - both options should maybe have the same prefix: google.bigtable.buffered.mutator.throttling